en/pe-docs/docs/devops/cd-pipelines/submit-and-manage-environment-promotion-approval-requests.md

```diff
@@ -0,0 +1,34 @@
+# Submit and Manage Environment Promotion Approval Requests
+
+In Choreo, when an administrator or platform engineer [configures the Environment Promotion workflow to require approval](../../governance/workflows/configure-approvals-for-choreo-workflows.md), you must submit a request to obtain approval to promote the deployment to a critical environment.
```

Comment:
```suggestion
If environment promotion workflows are configured for your organization, you must request and receive approval before deploying to a critical environment.
```

Reply:
`...for your organization`
Not sure if this is a organization wide configuration or pipeline specific one